src/System.Net.Requests/tests/FtpWebRequestTest.cs

```diff
@@ -70,7 +70,7 @@ public void GetResponse_ServerNameNotInDns_ThrowsWebException()
         public void GetResponse_ConnectFailure_ThrowsWebException()
         {
             // This is an invalid IP address, so we should fail to connect.
-            string serverUrl = "ftp://192.0.2.1/";
+            string serverUrl = "ftp://127.0.0.1/";
```

Comment:
> The test fails if FTP server is running on localhost. One of the solution is to add some random port number which is different then standard FTP port - 21

I don't recommend this: we need to avoid TCP simultaneous open issues now that all other tests rely on obtaining a random port from the OS.